### ISSUE TITLE
Roster Scraper: Calculate junior eligibility by age

### ISSUE THREAD

**Issue by xECK29x** [colinrsmall/ehm_roster_tools#22]
I would like to have contract end date calculated for specific major junior leagues (CHL mostly) where contract end date should be set to when the player would be last eligible to play (their overage year where they are 20 turning 21).  Would like to see this calculation supported if feasible.

**Comment by xECK29x** [comment]
Yes but its calculating end dates that would put the player over the age of 21, it is working based on years left (like college) but not on max age (AFAIK).

**Comment by colinrsmall** [comment]
Oh okay, I’ll take a look then.
